W-16350425-allowlisting-trusting-domains-fa (#351)
mulesoft/docs-access-management: modules/ROOT/pages/allowlisting-trusted-domains.adoc

[[auto-registration-domains]]
== Automatic Registration of Domains

> NOTE: If you have Exchange assets or Public Portals created before 2024, you need to manually register the domain.

Reviewer: ```suggestion
NOTE: You must manually register the domain for Exchange assets and public portals created before 2024.
```

When you publish an API specification to Exchange or send a request through the proxy, the base URI of the implementation is automatically added to the list of trusted domains.

> NOTE: Non-authenticated requests from public portals in Exchange or via shareable links in Mocking Service are automatically registered. 

Reviewer: ```suggestion
NOTE: Non-authenticated requests from public portals in Exchange or via shareable links in the Mocking Service are automatically registered. 
```
